content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -344,9 +344,7 @@ sometimes be started twice.
 If you do specify `.spec.parallelism` and `.spec.completions` both greater than 1, then there may be
 multiple pods running at once. Therefore, your pods must also be tolerant of concurrency.
 
-When the [feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
-`PodDisruptionConditions` and `JobPodFailurePolicy` are both enabled,
-and the `.spec.podFailurePolicy` field is set, the Job controller does not consider a terminating
+When the `.spec.podFailurePolicy` field is set, the Job controller does not consider a terminating
 Pod (a pod that has a `.metadata.deletionTimestamp` field set) as a failure until that Pod is
```

Comment:
I suppose `Once the job is finished, the Job status looks as follows:` should be also updated.

Reply:
> Maybe it would make sense to explain the FailureTarget under the Handling Pod and container failures.

Sure, I have expanded on this in the sibling PR: https://github.com/kubernetes/website/pull/46808

> Maybe as a new section as opposed to having it under Pod failure policy since its behavior is guarded by other 2 feature gates?

Yes, it is added in a new dedicated section in the sibling PR now.

> I suppose Once the job is finished, the Job status looks as follows: should be also updated.

Good catch, also updated there.